frontend/apps/reader/modules/readerfont.lua

```diff
@@ -405,6 +428,266 @@ function ReaderFont:onDecreaseFontSize(ges)
+                        g_font_family_fonts[family_tag] = nil
+                        self.font_family_fonts[family_tag] = nil
+                        self:updateFontFamilyFonts()
+                        if touchmenu_instance then touchmenu_instance:updateItems() end
+                    end,
+                    checked_func = function()
+                        if self.font_family_fonts[family_tag] == false then
+                            return true
+                        end
+                        return not self.font_family_fonts[family_tag] and not g_font_family_fonts[family_tag]
+                    end,
+                    separator = true,
+                },
+                max_per_page = 5,
+            },
+        }
+        for k, v in ipairs(face_list) do
+            local font_filename, font_faceindex, is_monospace, has_ot_math, has_emojis = cre.getFontFaceFilenameAndFaceIndex(v)
+            if i == 1 then
+                face_to_filename[v] = { font_filename, font_faceindex }
+            end
+            local ignore = false
+            if family_tag == "monospace" and not is_monospace then
+                ignore = true
+            end
+            if family_tag == "math" and not has_ot_math then
+                ignore = true
+            end
+            if family_tag == "emoji" and not has_emojis then
+                ignore = true
+            end
+            if not ignore then
+                table.insert(family_table.sub_item_table, {
+                    text_func = function()
+                        local text = v
+                        if font_filename and font_faceindex then
+                            text = FontList:getLocalizedFontName(font_filename, font_faceindex) or text
+                        end
+                        if g_font_family_fonts[family_tag] == v then
+                            text = text .. "   ★"
+                        end
+                        return text
+                    end,
+                    font_func = function(size)
+                        if G_reader_settings:nilOrTrue("font_menu_use_font_face") then
+                            if font_filename and font_faceindex then
+                                return Font:getFace(font_filename, size, font_faceindex)
+                            end
+                        end
+                    end,
+                    callback = function()
+                        if g_font_family_fonts[family_tag] == v then
+                            self.font_family_fonts[family_tag] = nil
+                        else
+                            self.font_family_fonts[family_tag] = v
+                            -- We don't use :notify() as we don't want this notification to be masked,
+                            -- to let the user know it's not global (so he has to use long-press)
+                            UIManager:show(Notification:new{ text = _("Family font set for this book only.") })
```

Comment:
`Font family` ;).

Reply:
Hmm, wait, I see what you were trying to say...

I'd possibly double down, then: `Font family font`?